pom.xml

```diff
@@ -104,7 +104,7 @@
 		<!-- plugin versions -->
 		<compiler.plugin.version>2.3.1</compiler.plugin.version>
 		<source.plugin.version>2.2.1</source.plugin.version>
-		<javadoc.plugin.version>2.9</javadoc.plugin.version>
+		<javadoc.plugin.version>3.0.0-M1</javadoc.plugin.version>
```

Comment:
curious about the reason for this one.

Reply:
@randallwhitman one of the travis builds was failing because of javadoc plugin version. I've updated it to the latest.